keras_cv/layers/preprocessing/base_image_augmentation_layer.py

```diff
@@ -293,33 +292,61 @@ def _augment(self, inputs):
         return result
 
     def _batch_augment(self, inputs):
-        return self._map_fn(self._augment, inputs)
+        augmented_batch = self._map_fn(self._augment, inputs)
+        return augmented_batch
 
     def _format_inputs(self, inputs):
+        metadata = {"is_dict": False, "use_targets": False}
         if tf.is_tensor(inputs):
             # single image input tensor
-            return {IMAGES: inputs}, False, False
+            metadata = {"is_dict": False, "use_targets": False}
+            inputs = {IMAGES: inputs}
+            return inputs, metadata
```

Comment:
reformulate the tf.is_tensor call as a guard clause.

All this means is make the next statement an `if`, and add a newline before it.